src/navigation/Nav.jsx

```diff
@@ -360,6 +364,17 @@ export class Nav extends React.Component {
 			);
 		});
 
+		const scriptLogoIcon = uxCapture ? (
+			<React.Fragment>
+				<UXCaptureEventMark mark="ux-image-onload-script-logo">
+					<img src={scriptLogo} alt={logo.logoAccessible} height="44px" />
+				</UXCaptureEventMark>
+				<UXCaptureInlineMark mark="ux-image-inline-script-logo" />
+			</React.Fragment>
+		) : (
+			<img src={scriptLogo} alt={logo.logoAccessible} height="44px" />
```

Comment:
Just a nitpick, but I would separate `<img>` into separate const and reuse it here and inside the fragment to make sure future updates don't need to be in two places.